website/content/en/docs/tasks/scheduling-pods.md

```diff
@@ -241,3 +241,58 @@ If instead the spread were 5, pods could be 5, 0, 0 or 3, 2, 0, or 2, 1, 2 and s
 * Karpenter is always able to improve skew by launching new nodes in the right zones. Therefore, `whenUnsatisfiable` does not change provisioning behavior.
 
 See [Pod Topology Spread Constraints](https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/) for details.
+
+## Persistent Volume Topology
+
+Karpenter automatically detects storage scheduling requirements and includes them in node launch decisions.
+
+In the following example, the `StorageClass` defines zonal topologies for `us-west-2a` and `us-west-2b` and binding mode `WaitForFirstConsumer`.
```

Comment:
do we support binding modes besides `WaitForFirstConsumer`?

Reply:
Yep. It follows the same steps, except the volume already exists.